src/portable/project.rs

```diff
@@ -1377,29 +1398,51 @@ pub fn info(options: &Info) -> anyhow::Result<()> {
                     println!("{}", instance_name);
                 }
             }
+            "cloud-profile" => {
+                if options.json {
+                    println!("{}", serde_json::to_string(&cloud_profile)?);
+                } else if let Some(profile) = cloud_profile {
+                    println!("{}", profile);
+                }
+            }
             _ => unreachable!(),
         }
     } else if options.json {
         println!("{}", serde_json::to_string_pretty(&JsonInfo {
             instance_name: &instance_name,
+            cloud_profile: cloud_profile.as_deref(),
             root: &root,
         })?);
     } else {
-        table::settings(&[
-            ("Instance name", &instance_name),
-            ("Project root", &root.display().to_string()),
-        ]);
+        let root = root.display().to_string();
+        let mut rows = vec![
+            ("Instance name", instance_name.as_str()),
+            ("Project root", root.as_str()),
+        ];
+        if let Some(profile) = cloud_profile.as_deref() {
+            rows.push(("Cloud profile", profile));
+        }
+        table::settings(rows.as_slice());
     }
     Ok(())
 }
 
+pub fn find_project_dirs_by_instance(name: &str) -> anyhow::Result<Vec<PathBuf>> {
+    find_project_dirs("instance-name", |val| name == val, true)
+        .map(|projects| projects.into_values().flatten().collect())
+}
+
 #[context("could not read project dir {:?}", stash_base())]
-pub fn find_project_dirs(name: &str) -> anyhow::Result<Vec<PathBuf>> {
-    let mut res = Vec::new();
+pub fn find_project_dirs(
+    get: &str,
+    f: impl Fn(&str) -> bool,
+    noisy: bool,
```

Comment:
```suggestion
    verbose: bool,
```
We usually use this.